res/values/strings.xml

```diff
@@ -426,7 +439,8 @@
     <string name="search">Search</string>
     <string name="search_explain">Search for chats, contacts, and messages</string>
     <string name="search_no_result_for_x">No results found for \"%s\"</string>
-
+    <string name="search_in_chat">Search in Chat</string>
+    <string name="exit_search">Exit Search</string>
```

Comment:
this is used as a hint for the "X" button beside search, right? a transator hint would probably be good in this case. intuitively, i'd also call this "End Search", there is nothing really "exited", the controls and the list stay.